mode/cypher/cypher.js

```diff
@@ -21,11 +21,11 @@
     var tokenBase = function(stream/*, state*/) {
       var ch = stream.next();
       if (ch ==='"') {
-        stream.match(/.*?"/);
+        stream.match(/^[^"]*"/);
```

Comment:
Because the `.*` "catch all" was ungreedy, this was looking for the first `"` in the string. We know there can not be another `"` before. The new pattern does the same, but is faster because it avoids any backtracking.